src/puppetlabs/ring_middleware/common.clj

```diff
@@ -48,6 +49,5 @@
                             http-opts)
                      request
                      prepare-cookies)]
-    (log/debug "Proxying request to" (:uri req) "to remote url" (str remote-uri)
-               ". Remote server responded with status" (:status response))
+    (log/debug (trs "Proxying request to {0} to remote url {1}. Remote server responded with status {2}" (:uri req) (str remote-uri) (:status response)))
```

Comment:
Ohhhh shoot I think I actually should break this one up into two strings. Larissa said that having one sentence per string would make it more cost efficient (i18n library will be able to find repeated sentences so that we can send on sentence to a translator and use it multiple times). I can add a newline between the two sentences if you think that would be good.